lib/appender/standard.js

```diff
@@ -111,8 +111,13 @@ function standardLogAppender(level, exception, message, data) {
   // If stack is not defined we are in a browser that does not fully support our logging.
   // Or one of our custom loggers was called, e.g. Logger.debug('message').
   if (exception === undefined || exception === null || exception.stack === null) {
-    var scripts = window.document.getElementsByTagName('script');
-    scriptURL = (window.document.currentScript || scripts[scripts.length - 1]).src;
+    if (typeof window.document.getElementsByTagName === 'function') {
+      var scripts = window.document.getElementsByTagName('script');
+      scriptURL = (window.document.currentScript || scripts[scripts.length - 1]).src;
+    } else {
+      // Probably not in a browser, return unknown log.
+      scriptURL = Canadarm.constant.UNKNOWN_LOG;
```

Comment:
We know why this is not here, I'd rather not set the scriptURL at all because it does not make sense within the node context, unless we set it to the file path.